appserver/payara-appserver-modules/jaxrs-client-tracing/src/main/java/fish/payara/requesttracing/jaxrs/client/JaxrsClientRequestTracingFilter.java

```diff
@@ -175,6 +177,14 @@ public void filter(ClientRequestContext requestContext, ClientResponseContext re
                     serviceLocator.getService(InvocationManager.class)))
                     .scopeManager().active()) {
                 
+                if (activeScope == null) {
+                    // occurs when request tracing has been enabled between during method invocation
+                    //in this case nothing will be done, but log in case of other error
+                    Logger.getLogger(JaxrsClientRequestTracingFilter.class.getName()).log(Level.FINE,
+                            "activeScope in  opentracing request tracing filter was null for {0}", responseContext);
+                    return;
```

Comment:
This is a very aggressive solution - I want to give this a bit of a hammering before merging.

Reply:
The comment also has grammatical and formatting errors, and isn't entirely accurate (you can also get here if the OpenTracing context was somehow lost)